src/hamster/lib/__init__.py

```diff
@@ -23,7 +23,7 @@
 tag_re = re.compile(r"""
     [\s,]*     # any spaces or commas (or nothing)
     \#          # hash character
-    ([^#\s]+)  # the tag (anything but # or spaces)
+    ([^#\s,]+)  # the tag (anything but #, spaces or comma)
```

Comment:
This change would fix the strange behaviour with `#456@cat`:
```
    ([^#@\s,]+)  # the tag (anything but #, @ederag , spaces or comma)
```

Reply:
`([^#@\s,]+)  # the tag (anything but #, @ , spaces or comma)`

Reply:
Wouldn't that forbid '@' in tags and be a regression ?
IMO the behavior is certainly puzzling at first sight, but clearly understandable
if the rules can be kept simple.

Reply:
Ah, so we also need to allow @ in the middle of tags. I never would have imagined that. Okay.